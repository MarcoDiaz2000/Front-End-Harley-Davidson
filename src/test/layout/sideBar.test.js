import React from 'react';
import renderer from 'react-test-renderer';
import SideBar from '../../components/__mock__/Sidebar';

describe('MotorItem', () => {
  it('renders correctly', () => {
    const component = renderer.create(<SideBar />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
