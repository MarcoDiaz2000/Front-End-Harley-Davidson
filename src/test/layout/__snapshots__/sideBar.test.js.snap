// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MotorItem renders correctly 1`] = `
<div
  className="flex flex-col gap-12 sticky top-0 h-screen border-r border-customBorder font-roboto bg-bodyBg"
>
  <div
    className=" w-full justify-center items-center mt-4 mb-8 hidden md:flex"
  >
    <div
      className="w-40"
    >
      <img
        alt="Harley Davidson Logo"
        className="h-auto w-full"
        src="src/img"
      />
    </div>
  </div>
  <nav
    className="flex flex-col"
  >
    <button
      className="hover:text-white hover:bg-customYellow pt-2 pb-2 font-medium md:hidden"
      type="button"
    >
      menu
    </button>
    <div
      className="hidden md:flex flex-col"
    >
      <a
        className="hover:text-white text-white hover:bg-customYellow pt-2 pb-2 pl-6 ml-4 font-medium"
        href="/motors"
      >
        <span
          className="mr-2 text-white "
        >
          icon
        </span>
        MOTORS
      </a>
      <a
        className="hover:text-white text-white hover:bg-customYellow pt-2 pb-2 pl-6 ml-4 font-medium"
        href="/reservations"
      >
        <span
          className="mr-2 text-white "
        >
          icon
        </span>
        My Reservatios
      </a>
      <a
        className="hover:text-white text-white hover:bg-customYellow pt-2 pb-2 pl-6 ml-4 font-medium"
        href="/reserve"
      >
        <span
          className="mr-2 text-white "
        >
          icon
        </span>
        Reserve
      </a>
      <a
        className="hover:text-white text-white hover:bg-customYellow pt-2 pb-2 pl-6 ml-4 font-medium"
        href="/add"
      >
        <span
          className="mr-2 text-white "
        >
          icon
        </span>
        ADD MOTOR
      </a>
      <a
        className="hover:text-white text-white hover:bg-customYellow pt-2 pb-2 pl-6 ml-4 font-medium"
        href="/delete"
      >
        <span
          className="mr-2 text-white "
        >
          icon
        </span>
        DELETE
      </a>
    </div>
  </nav>
  <div
    className="flex justify-center text-white mt-auto pb-4 hidden md:flex"
  >
    <span>
      icon
    </span>
    <span>
      icon
    </span>
    <span>
      icon
    </span>
  </div>
</div>
`;
